src/client/AM.PA.MonitoringTool/ActivityMapper/ActivityCategoryExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Linq;
```

Comment:
Why does this file not have the license notice as the one above?

Reply:
I would prefer to stay semi-anonymously. I added a license notice without a name. You're free to use yours.

Reply:
This might be a question for @casaout: What is your policy on license headers? Should every file have one?